src/main/java/cs/f10/t1/nursetraverse/logic/commands/ExportCommand.java

```diff
@@ -1,39 +1,38 @@
-package seedu.address.logic.commands;
+package cs.f10.t1.nursetraverse.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_FILENAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEXES;
 
 import java.io.IOException;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
-import seedu.address.commons.core.Messages;
-import seedu.address.commons.core.index.Index;
-import seedu.address.importexport.CsvUtil;
-import seedu.address.importexport.exceptions.ExportingException;
-import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.Model;
-import seedu.address.model.person.Person;
+import cs.f10.t1.nursetraverse.commons.core.Messages;
+import cs.f10.t1.nursetraverse.commons.core.index.Index;
+import cs.f10.t1.nursetraverse.importexport.CsvUtil;
+import cs.f10.t1.nursetraverse.importexport.exceptions.ExportingException;
+import cs.f10.t1.nursetraverse.logic.commands.exceptions.CommandException;
+import cs.f10.t1.nursetraverse.logic.parser.CliSyntax;
+import cs.f10.t1.nursetraverse.model.Model;
+import cs.f10.t1.nursetraverse.model.patient.Patient;
 
 /**
- * Exports Person data from the AddressBook
+ * Exports Patient data from the PatientBook
  */
 
 public class ExportCommand extends Command {
 
-    public static final String COMMAND_WORD = "export";
+    public static final String COMMAND_WORD = "app-export";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Exports persons data into a .csv file in /exports.\n"
-            + "Persons are exported selectively by index.\n"
-            + "If indexes are not provided, all persons will be exported.\n"
+            + ": Exports patients data into a .csv file in /exports.\n"
+            + "Patients are exported selectively by index.\n"
+            + "If indexes are not provided, all patients will be exported.\n"
             + "File name must be new - overriding an existing file is not permitted.\n"
-            + "Parameters: [" + PREFIX_FILENAME + "FILENAME] "
-            + "[" + PREFIX_INDEXES + "INDEXES]...\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_FILENAME + "patient_data "
-            + PREFIX_INDEXES + "2 " + PREFIX_INDEXES + "4 " + PREFIX_INDEXES + "6";
+            + "Parameters: [" + CliSyntax.PREFIX_FILENAME + "FILENAME] "
+            + "[" + CliSyntax.PREFIX_INDEXES + "INDEXES]...\n"
+            + "Example: " + COMMAND_WORD + " " + CliSyntax.PREFIX_FILENAME + "patient_data "
+            + CliSyntax.PREFIX_INDEXES + "2 " + CliSyntax.PREFIX_INDEXES + "4 " + CliSyntax.PREFIX_INDEXES + "6";
```

Comment:
Can import each of these in this file so that they can be used directly by name (i.e. `PREFIX_FILENAME` instead of `CliSyntax.PREFIX_FILENAME`).